util/find-include-nits

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env perl
+# Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+# Run this program like this:
+#       perl util/err-to-raise [files...]
```

Comment:
You might want to edit this

Reply:
cut and paste is my very best friend ever :)
updated commit pushed.